### PR TITLE
Created README.md

### PR DESCRIPTION
**Pull request by Giftea**
This PR introduces a comprehensive README file for the HERA Mobile App repository. The README provides essential information for users and contributors, outlining the project's mission, features, and setup instructions.

**Changes Made:**

- Added a welcome message highlighting HERA's mission to empower local health organisations and improve healthcare access for refugees.
- Included an overview detailing the app's purpose, initial focus on maternal and child health, and the benefits of open-source technology.
- Provided step-by-step instructions for getting started, including prerequisites and installation guidance.
- Added community support resources.
- Included acknowledgments and a license section.

### CODE REVIEW

**Comment by Giftea** [comment]
@momocentral Please review, thanks.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# HERA Mobile App
+
+Welcome to the HERA Mobile App repository! HERA is dedicated to connecting refugee communities with essential healthcare services through an open-source mobile health platform. By making our technology freely available, we hope to empower organizations like NGOs and local aid groups to customize and deploy healthcare solutions that meet the specific needs of refugees in their regions.
+
+## Overview
+
+HERA’s mission is to support vulnerable populations, particularly refugees, by increasing access to vital healthcare services. As part of this mission, HERA focuses on preventive healthcare, with an initial goal to improve prenatal care and vaccination rates among refugees. This project began by addressing critical healthcare needs among Syrian refugees in Turkey, and HERA plans to extend support to refugees globally.
+
+Our technology stack includes a React Native mobile app, a React.js and Django web backend with robust APIs, and an admin panel, all of which can be adapted by organizations to provide tailored healthcare solutions.
```
Comment (husam79) [inline README.md]: We don't use React in the mobile app, just React Native. For the backend Django app, please, provide a direct link to the backend repo.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- **Mobile Health Platform**: Connects refugees with available healthcare services.
+- **Customizable Solution**: Open-source codebase that NGOs and organizations can adapt to specific regions and needs.
+- **Focus on Maternal and Child Health**: Prioritizes prenatal care and childhood vaccinations.
+  
+## Getting Started
+
+To set up the HERA Mobile App locally, follow these steps:
+
+### Prerequisites
+- [Node.js](https://nodejs.org/en/download/) (version 16.0 or higher)
```
Comment (husam79) [inline README.md]: I haven't tested it on a version higher than 16 yet.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- [npm](https://www.npmjs.com/get-npm) or [yarn](https://yarnpkg.com/getting-started/install)
+- [Android Studio](https://developer.android.com/studio) (for Android development) or [Xcode](https://developer.apple.com/xcode/) (for iOS development)
+
+### Installation
```
Comment (husam79) [inline README.md:25]: Check the document I have shared with you for further details

File: README.md (hunk 2 above)
Comment (Giftea) [reply README.md]: What node version is preffered?

Comment (husam79) [reply README.md]: 16.20.2

**Pull request merged**